config/src/lib.rs

```diff
@@ -110,6 +115,10 @@ impl ConnectionsConfig {
     pub fn default_known_peers() -> Vec<SocketAddr> {
         Vec::default()
     }
+
+    fn default_storage_peers_period() -> Duration {
+        Duration::from_secs(30)
```

Comment:
This is a magic number, please:
- Move it to a more visible spot
- Make it static
- Add a comment for it, potentially reasoning about considerations when deciding the value